fiftyone/core/collections.py

```diff
@@ -599,46 +609,55 @@ def count_sample_tags(self):
         """
         return self.count_values("tags")
 
-    def tag_labels(self, tag, label_fields=None):
-        """Adds the tag to all labels in the specified label field(s) of this
-        collection, if necessary.
+    def tag_labels(self, tags, label_fields=None):
+        """Adds the tag(s) to all labels in the specified label field(s) of
+        this collection, if necessary.
 
         Args:
-            tag: a tag
+            tags: a tag or iterable of tags
             label_fields (None): an optional name or iterable of names of
                 :class:`fiftyone.core.labels.Label` fields. By default, all
                 label fields are used
         """
+        if etau.is_str(tags):
+            tags = [tags]
+        else:
+            tags = list(tags)
 
-        def _add_tag(tags):
-            if not tags:
-                return [tag]
-
-            if tag in tags:
+        def _add_tags(_tags):
+            if not _tags:  # handles None
```

Comment:
I was allowing for `tags=None`, but I suppose that couldn't actually happen since the data model defaults to `[]` in those cases.

I was getting errors without checks like this in other contexts like `set_values()` where missing data would lead to `None` in places that would otherwise be lists.

I think I'll leave the checks here just to be safe.

Reply:
Yes, I bring it up just because I was seeing some weirdness today in my work. Classifications showing up with only `tags` present (no `label`, etc.) in server responses. Something I need resolve.

Reply:
Oooh that might be something that `tag_labels()` does not handle- cases where a sample has no `Classification`...

I fixed some related things on https://github.com/voxel51/fiftyone/pull/911 but may need to do a bit more. I'll investigate

Reply:
ok `None`-valued labels were definitely an issue, but I've fixed that on this branch now.